teerex/src/benchmarking.rs

```diff
@@ -87,7 +78,6 @@ benchmarks! {
 	// Benchmark `register_tcb_info` with the worst possible conditions:
 	// * tcb registration succeeds
 	register_tcb_info {
-		ensure_not_skipping_ra_check();
```

Comment:
Need to make sure that the benchmarks runtime does not skip this otherwise, we will get weights, which are orders of magnitude weaker. To make sure that the benchmark is correct, we should probably overwrite the fields here, regardless of what is put in the genesis config: https://github.com/integritee-network/pallets/issues/219

Reply:
I see your point